swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else: 
+            return False
```

Comment:
Consider flipping the sense of the condition around to handle the special case first (something is falsy) quickly and get out, rather than having that dangling return at the end. Notice that it lets us move the main focus of the function out of an indented block, letting it stand out more clearly.
```python
    if item not in self.inventory:
        return False

    self.inventory.remove(item)
    return item
```